browser-extension/addTournesolRecommendations.js

```diff
@@ -3,8 +3,16 @@
 // This part is called on connection for the first time on youtube.com/*
 /* ********************************************************************* */
 
+const defaultVideoNumber = 4;
```

Comment:
This constant defines the number of videos per row, and not only the number of visible videos by default. So `videosPerRow` may be a better name ?